scripts/deploy-and-test-on-instance.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -eu
+
+JOB_UPDATE_STAGING="8d452c26-5a39-4162-b3b8-4da5efdf0b76"
+
+RUNDECK_TOKEN='2hE42p5phY1qSJ0YJUch2AqnYbXrhDec'
+INSTANCE_TESTCAFE='testcafe.cozy.rocks'
```

Comment:
I think this line and the next 2 need to me removed, no?

Reply:
Indeed....